README.md

```diff
@@ -1,112 +1,113 @@
 # PromptSource
-Promptsource is a toolkit for collecting and applying prompts to NLP datasets.
+**PromptSource is a toolkit for creating, sharing and using natural language prompts.**
 
-Promptsource uses a simple templating language to programatically map an example of a dataset into a text input and a text target.
+Prompts are functions that map an example from a dataset to a natural language input and target output PromptSource contains a growing collection of prompts (which we call **P3**: **P**ublic **P**ool of **P**rompts). As of January 20, 2022, there are ~2'000 English prompts for 170+ English datasets in [P3](https://huggingface.co/datasets/bigscience/P3).
 
```

Comment:
I think some popular language here about how prompting is really important because of GPT-3 and then T0 / FLAN, and how it is a new type of NLP tool with lots of interest etc. Would be really nice.